Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -460,12 +459,53 @@ protected class ClassWriter : IClassWriter {
       if (!createBody) { return null; }
       if (isStatic || NeedsCustomReceiver(member)) { wr.WriteLine("@staticmethod"); }
       wr.Write($"def {name}(");
-      WriteFormals(formals, isStatic && !NeedsCustomReceiver(member), wr);
+      var sep = "";
+      WriteRuntimeTypeDescriptorsFormals(member, ForTypeDescriptors(typeArgs, member, lookasideBody), wr, ref sep, tp => tp.CompileName);
+      WriteFormals(formals, isStatic && !NeedsCustomReceiver(member), ref sep, wr);
       return wr.NewBlockPy("):", close: BlockStyle.Newline);
     }
 
     protected override string TypeDescriptor(Type type, ConcreteSyntaxTree wr, IToken tok) {
-      throw new UnsupportedFeatureException(Token.NoToken, Feature.RuntimeTypeDescriptors);
+      Contract.Requires(type != null);
+      Contract.Requires(tok != null);
+      Contract.Requires(wr != null);
+
```

Comment:
Perhaps worth a comment here that our representation of a type descriptor is just a lambda that produces the default value (as opposed to a separate type as in at least some target languages)